config/express.js

```diff
@@ -64,6 +64,7 @@ module.exports = (app, config) => {
         ],
         fontSrc: [
           'assets.nhs.uk',
```

Comment:
This could be changed to `'*.nhs.uk'` to avoid having multiple entries.

Reply:
Changed.